packages/client/src/features/tools/node-creation/container-manager.ts

```diff
@@ -44,21 +44,36 @@ export interface TrackedInsert {
     options: InsertOptions;
 }
 
+export interface IContainerManager {
```

Comment:
Do we expect alternative implementations of this interface? I'm just wondering because for some manager classes we do not do the interface+symbol combination, e.g., `ChangeBoundsManager` and `GridManager` whereas for others we do, e.g., `IHelperLineManager`, `IToolManager`.

I wonder if we have any guidelines for it @tortmayr ?

Reply:
I extracted the interface to allow for custom `ContainerManager` implementations without having to modify the `NodeCreationTool`.

Reply:
I don't think we have a guideline for that. We adopted the pattern from sprotty and even there its not consistent and sometime interface+symbol are used for injection, othertimes the class  is injected directly.

I don't have a strong opinion on this but if we go the route of the extracted interface we should make it at least consistent for the other manager classes and also bind them via interface.

Reply:
Fine with me, we can do it as part of this PR or open another issue to align this more consistently. I don't think we will run into any backwards compatibility issues when we do the binding properly.

Reply:
Sure, I have created a follow-up issue for the remaining *Manager classes and will tackle this right after this PR then.
See https://github.com/eclipse-glsp/glsp/issues/1393

Reply:
Works for me, thank you very much!